zk-sdk/src/zk_elgamal_proof_program/proof_data/batched_grouped_ciphertext_validity/handles_2.rs

```diff
@@ -150,6 +153,13 @@ impl BatchedGroupedCiphertext2HandlesValidityProofData {
     pub fn to_bytes(&self) -> Box<[u8]> {
         bytes_of(self).into()
     }
+
+    #[cfg_attr(target_arch = "wasm32", wasm_bindgen(js_name = fromBytes))]
+    pub fn from_bytes(bytes: &[u8]) -> Result<Self, ProofDataError> {
+        bytemuck::try_from_bytes(bytes)
+            .copied()
+            .map_err(|_| ProofDataError::Deserialization)
+    }
```

Comment:
Is it possible to use the macro for these too?

Reply:
Yes, sorry it seems like I screwed up the git commits and did not include the changes in the rest of the files...